components/style/themes/default.less

```diff
@@ -384,7 +384,18 @@
 // ---
 @tabs-card-head-background: @background-color-light;
 @tabs-card-height: 40px;
+@tabs-card-active-color: @primary-color;
 @tabs-title-font-size: @font-size-base;
+@tabs-ink-bar-bg-color: @primary-color;
+@tab-bar-margin: 0 0 16px 0;
```

Comment:
should be 4 different vars? there 4 different tab bar positions.

Reply:
`@tab-bar-margin` was just [moved from here](https://github.com/ant-design/ant-design/blob/master/components/tabs/style/index.less#L7) and remains the same. This PR just aims to enable altering the key colors of the `<Tab />` component using the theme file.